data/glsk/glsk-document-api/src/main/java/com/farao_community/farao/data/glsk/api/Chronology.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.glsk.api;
+
+import java.time.Instant;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public interface Chronology<I> {
```

Comment:
Why not using DatChronology from farao-commons ?

Reply:
I think for two reasons:
- DataChronology interface is a bit too permissive with construction methods
- It is not really smooth for user if you combine it with GlskProvider or ScalableProvider. Maybe I forget Chronology<I extends Chronology<I>>, the idea was to be able to enhance any interface with a notion of time, so here with any other interface you can do interface.selectInstant(instant).interfaceMethod() which I find quite user friendly.

Reply:
Perhaps you could modify a little bit the DataChronology interface to fit with your needs ? One Simple chronology (equivalent to yours in a way) extended by a ReadWriteOne (DataChronology) and put all of them in the chronology package...
Here I do not like that we have a feature available in a dedicated package, but we do not use it and create an equivalent feature in another package ....

Reply:
And for me, I cannot see where is the difference (semantically) between Chronolgy::selectInstant and DataChronology::getDataForInstant ....

Reply:
I agree so I adapted according to what you said. I also added a refactoring with a ZonalData<I> interface which enables to generalize and reduce numbers of implementations.